src/em_pre.js

```diff
@@ -0,0 +1,10 @@
+var Module = {}
+
+Module['onAbort'] = function(reason) {
+  // d8 does not exit with status 1 when a promise throws. Emscripten calls
```

Comment:
might want to say  'JS shells' instead of just 'd8' and maybe also mention both throws and rejections (since they are equivalent for most purposes)